twitter_tokens.txt

```diff
@@ -0,0 +1,4 @@
+consumer_key: H00AP00nuAyzpNtPDy2lsG5kt
```

Comment:
@natjoe4 Make sure you add file names that have sensitive information like keys so that they are not commited to github. for example in this case in the .gitignore it would be adding `twitter_tokens.txt`